hack/control-plane.sh

```diff
@@ -21,6 +21,6 @@ readonly CONTROL_PLANE_POST_INSTALL_CONFIG_DIR=control-plane/config/post-install
 
 # Note: do not change this function name, it's used during releases.
 function control_plane_setup() {
-  ko resolve ${KO_FLAGS} -Rf "${CONTROL_PLANE_CONFIG_DIR}" | "${LABEL_YAML_CMD[@]}" >>"${EVENTING_KAFKA_CONTROL_PLANE_ARTIFACT}" || return $?
```

Comment:
was the append wrong?

Reply:
Yes, the root cause is that we weren't deleting the post-install artifact before re-building, so locally when iterating I was getting duplicate content in the same file.

Since we generate YAMLs in one command we don't need to append at all.